interface/super/edit_layout.php

```diff
@@ -565,7 +565,7 @@ class='optin noselect' style='width:100%' />";
 
         echo "<input type='text' name='fld[$fld_line_no][list_id]'  id='fld[$fld_line_no][list_id]' value='" .
         htmlspecialchars($linedata['list_id'], ENT_QUOTES) . "'".$type.
-        " size='6' maxlength='30' class='optin listid' style='width:100%;cursor:pointer'".
+        " size='6' maxlength='100' class='optin listid' style='width:100%;cursor:pointer'".
```

Comment:
I think it's necessary because if option_id could be 100 it's possible to create new list with list_id bigger than 31 ('lists' is also list in the list_options table and the id of every list is saved in option_id column)